js/web/lib/wasm/wasm-training-core-impl.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {InferenceSession} from 'onnxruntime-common';
+
+import {SerializableModeldata, SerializableSessionMetadata} from './proxy-messages';
+import {setSessionOptions} from './session-options';
+import {getInstance} from './wasm-factory';
+import {checkLastError} from './wasm-utils';
+
+const NO_TRAIN_FUNCS_MSG = 'Built without training APIs enabled. ' +
+    'Make sure to use the onnxruntime-training package for training functionality.';
+
+export const createCheckpointHandle = (checkpointData: SerializableModeldata): number => {
+  const wasm = getInstance();
+
+  let checkpointHandle = 0;
+
+  try {
+    if (wasm._OrtTrainingLoadCheckpoint) {
+      checkpointHandle = wasm._OrtTrainingLoadCheckpoint(checkpointData[0], checkpointData[1]);
```

Comment:
what is : "checkpointData[0], checkpointData[1]"

Reply:
the checkpointData buffer & the length of checkpointData -- updated the code to hopefully be more readable